module/VuFind/src/VuFind/Export.php

```diff
@@ -31,6 +31,7 @@
 
 use Laminas\View\Renderer\PhpRenderer;
 use VuFind\Config\Config;
+use VuFind\RecordDriver\AbstractBase;
```

Comment:
Might be more clear to import `as RecordDriver` since a typehint of `AbstractBase` is pretty useless (for a human reader, at least).

Reply:
Done!